media-picker/build.gradle

```diff
@@ -18,15 +18,21 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    repositories {
+        jcenter()
+        maven { url 'https://jitpack.io' }
+    }
 }
 
 dependencies {
 
-    //noinspection GradleDynamicVersion || File provider compatibility.
-    compile 'com.android.support:support-core-utils:25.+'
+    // File provider compatibility.
+    compile 'com.android.support:support-core-utils:25.3.1'
 
-    //noinspection GradleDynamicVersion || File cropping utility.
-    compile 'me.villani.lorenzo.android:android-cropimage:1.+'
+    // File cropping utility.
+    compile 'com.github.yalantis:ucrop:2.2.1'
+    compile 'com.github.yalantis:ucrop:2.2.1-native'
```

Comment:
Yes that would be a better idea. I thought I needed this but actually didn't (was specifying bad params) and never got rid of it.